packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -31,12 +31,18 @@ TableColumnTextColumnConfig
 
     public readonly cellTemplate = cellTemplate;
 
-    public getColumnConfig(): TableColumnTextColumnConfig {
-        return { placeholder: this.placeholder ?? '' };
+    protected fieldNameChanged(
+        _prev: string | undefined,
+        _next: string | undefined
```

Comment:
nit: Think I prefer (kinda arbitrarily) to just leave these parameters off if unused for the `*Changed` methods. They aren't coupled to any actual type so if the parameters actually do change the parameters as-is would just be misleading.

Would be nice to actually have them be well-typed, could check if FAST is exposing a type to help do that, but didn't see anything from a quick glance.

Reply:
Alright, got the itch and did some [type shenanigans](https://www.typescriptlang.org/play?#code/PTAEFcBcEsBsYKYGcB0kkCgOQJ4AcFQBJAO0gQCcSBDWAeQCMlKA3SpAHgAUKB7AioiQA+UAF5QAbwyhZoANoBrBDlDQSoZTl4AzUD36UhoAGSgkkCuoDmoaklAADACSStAXwDCAC2olrCAAmjgC6APwAXKAAFLywgQBqtOAIUQYCQkoqIaAAPhAkgQg66kEANKAkCADuSbApaXwZ0MhZODn54IXFpYEAlOKiLLzQgQDcGO5YIKAMvAxomADGsPYOAELzagC2eLAI2whkDqTkVLSMzBRsFJxc0EuKHJsMFQDkLLRvwqLScqB4cAMWAPUCfWCRcyWGwTf4zQHA0Hgnx+AKBaIDSRTOFgBEgpZg2go-xBaLUKIWKz+PIFIolKr9KTYuR4pFE3wk9HkqFU2ydbr08qzCnQ6n8um9TFTKZAA)

Get to opt-in which properties will have a changed handler and  it figures out and enforces the type signature :)

Reply:
That's pretty cool!

I'd prefer not to include that change as part of this PR. Should we create a tech debt item to switch everything over to the typing that you got working?

Reply:
Yea absolutely not for this one! A techdebt issue sounds good 👍

Reply:
I've created #1068